server/sonar-process-monitor/src/main/java/org/sonar/application/cluster/AppStateClusterImpl.java

```diff
@@ -30,6 +30,7 @@
 import org.sonar.application.AppStateListener;
 import org.sonar.application.config.AppSettings;
 import org.sonar.process.ProcessId;
+import org.sonar.process.ProcessProperties;
```

Comment:
as you are just stating yourself, one commit for the UUID and one for the cluster. `AppStateClusterImpl` has nothing to do with the UUID